doc/fwutil/fwutil.md

```diff
@@ -195,16 +214,20 @@ fwutil
 |              |--- fw -y|--yes <fw_path>
 |
 |--- update
-     |--- chassis
-     |    |--- component <component_name>
-     |         |--- fw -y|--yes -f|--force -i|--image=<current|next>
-     |
-     |--- module <module_name>
-          |--- component <component_name>
-               |--- fw -y|--yes -f|--force -i|--image=<current|next>
-```
+|    |--- chassis
+|    |    |--- component <component_name>
+|    |         |--- fw -y|--yes -f|--force -i|--image=<current|next>
+|    |
+|    |--- module <module_name>
+|         |--- component <component_name>
+|              |--- fw -y|--yes -f|--force -i|--image=<current|next>
+|
+|--- auto_update
+     |--- fw -i|--image=<current|next> --b|--boot=<any|none|fast|warm|cold|powercycle>
+     |--- fw -z|--fw-image=<fw_package.tar.gz> --b|--boot=<any|none|fast|warm|cold|powercycle>
```

Comment:
@sujinmkang How do you expect to handle power cycle scenario? SONiC can only distinguish between cold/fast/warm reboots.

Reply:
that is good question. I think we should limit this boot option to non/fast/warm/cold.